source/templates/handlebars-basics.md

```diff
@@ -1,64 +1,72 @@
 Ember.js uses the [Handlebars templating library](http://www.handlebarsjs.com)
-to power your app's user interface. Handlebars templates are just like
-regular HTML, but also give you the ability to embed expressions that
-change what is displayed.
+to power your app's user interface. Handlebars templates contain static HTML and
+dynamic content inside what we call mustaches: `{{}}`.
```

Comment:
Is "mustache" really the term we want to use? It's not currently used anywhere in the Guides. Maybe "double curly braces"? That term is used elsewhere already.